scripts/test/test-backend.sh

```
@@ -11,6 +16,23 @@ RED='\033[0;31m'
+NODE_B_PORT="${GHOSTWIRE_NODE_B_PORT:-22}"
+
+# SSH options (use identity file from SSH config)
+SSH_OPTS="-o StrictHostKeyChecking=ask -o ConnectTimeout=5 -o BatchMode=yes"
+
+# Function to run SSH command on Node B
+ssh_node_b() {
+    if [ -z "$NODE_B_HOST" ]; then
+        echo -e "${YELLOW}Warning: NODE_B_HOST not configured, skipping remote command${NC}"
+        return 1
+    fi
+    ssh $SSH_OPTS -p "$NODE_B_PORT" "${NODE_B_USER}@${NODE_B_HOST}" "$@"
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)